apps/server/src/app/export/msword.ts

```diff
@@ -40,6 +58,225 @@ export class MSWordExporter {
         text: implementationGuide.title || implementationGuide.name,
         heading: HeadingLevel.HEADING_1
       }));
+
+
+      this.body.push(new Paragraph({
+        text: "Description:" + implementationGuide.description
+      }));
+      this.body.push(new Paragraph({
+        text: "Publisher:" + implementationGuide.publisher
+      }));
+      this.body.push(new Paragraph({
+        text: "Copyright:" + implementationGuide.copyright
+      }));
+
+      this.body.push(new Paragraph({
+        text: ""
+      }));
+
+      bundle.entry
+        .filter(entry => entry.resource && (entry.resource.resourceType === 'StructureDefinition'))
+        .forEach((entry) => {
+          const extensionUrlKeys = Object.keys(Globals.extensionUrls);
+          const extensionUrls = extensionUrlKeys.map((key) => Globals.extensionUrls[key]);
+          const keys = Object.keys(entry.resource);
+
+          this.logger.log(" VERSION:" + implementationGuide.version);
+          this.logger.log(" FHIR VERSION:" + implementationGuide.fhirVersion);
+
+          const structureDefinition: STU3StructureDefinition | R4StructureDefinition = <STU3StructureDefinition | R4StructureDefinition> entry.resource;
+
+
+            this.logger.log(" STRUCTURE name:" + structureDefinition.name);
+            this.logger.log(" STRUCTURE purpose :" + structureDefinition.purpose);
+            this.logger.log(" STRUCTURE title :" + structureDefinition.title);
+
+
+            this.logger.log(" STRUCTURE version:" + structureDefinition.version);
+
+            this.logger.log(" STRUCTURE fhirversion:" + structureDefinition.fhirVersion);
+
+          this.logger.log(" STRUCTURE publisher:" + structureDefinition.publisher);
+
+          this.logger.log(" STRUCTURE contact:" + structureDefinition.contact);
+
+          this.logger.log(" STRUCTURE copyright:" + structureDefinition.copyright);
+
+
+          // const structureDefinition: StructureDefinition = <StructureDefinition> entry.resource;  // we can do this because we know entry.resource will be a StructureDefinition because of the filter
+
+          this.body.push(new Paragraph({ text: `Name: ${structureDefinition.name}` }));    // name is required, so we can expect it to be populated with a value
+          if (structureDefinition.title) {
+            this.body.push(new Paragraph({ text: `Title: ${structureDefinition.title}` }));    // name is required, so we can expect it to be populated with a value
+          }
+
+
+          if (structureDefinition.differential) {
+            (structureDefinition.differential.element || []).forEach((element, eIndex) => {
+                if(element.id === 'Extension'){
```

Comment:
Why `if (element.id === 'Extension') {`?
Seems to me the logic can be the same regardless of the value of element.id